CodeTrans/docker/ui/svelte/src/lib/shared/constant.ts

```diff
@@ -17,22 +17,22 @@ import type { Language } from "../../routes/types.js";
 export const languagesList: Language[] = [
   { name: "C" },
   { name: "C++" },
-  { name: "C#" },
+  // { name: "C#" },
```

Comment:
Remove the comments if they are not used.

Reply:
Each language has its own way of highlighting,we can keep it first, and then use it directly when the backend supports it.